cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
Is this code that is now part of Kubernetes (i.e. being donated by the authors/copyright holders) or code that we're forking and tweaking?

The Copyright line is pretty much legally meaningless, but it's required nonetheless.